pymoto/modules/assembly.py

```diff
@@ -223,6 +223,42 @@ def _prepare(self, domain: DomainDefinition, *args,
         super()._prepare(domain, self.KE, *args, **kwargs)
 
 
+def ConsistentMassEq(domain: DomainDefinition, ndof: int, MP: float = 1.0):
```

Comment:
Probably the old AssembleMass can be replaced by this one. Instead of having 2 Mass assembly modules

Reply:
Guess this will fix itself when I change AssembleMass to its generalized version